ci/buildspec.yml

```
@@ -1,18 +1,41 @@
 version: 0.2
-
+env:
+  variables:
+    IS_GENERIC_IMAGE: "True"
+    CODEBUILD_RESOLVED_SOURCE_VERSION: "sparkml-v33"
+    TGT_IMAGE: "515193369038.dkr.ecr.us-west-2.amazonaws.com/sagemaker-sparkml-serving:3.3"
+    DLC_IMAGES: "515193369038.dkr.ecr.us-west-2.amazonaws.com/sagemaker-sparkml-serving:3.3-pre-scan"
 phases:
+  install:
+    runtime-versions:
+      python: latest
+    commands:
+      - pip3 install pytest
+      - pip3 install -r $CODEBUILD_SRC_DIR_Source2/src/requirements.txt
+      - pip3 install -r $CODEBUILD_SRC_DIR_Source2/test/requirements.txt
   pre_build:
     commands:
     - echo Logging in to Amazon ECR...
     - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
```

Please change this command from `get-login` to `get-login-password`, which is more secure.

Example:
```suggestion
    - aws ecr get-login-password --region $AWS_DEFAULT_REGION | docker login --username AWS --password-stdin 515193369038.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com
```